src/app/authentication/components/form-login/form-login.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-form-login',
+  templateUrl: './form-login.component.html',
+  styleUrls: ['./form-login.component.scss']
+})
+export class FormLoginComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
```

Comment:
@carryall Could we remove this extra space?

Reply:
@junan Same as [this comment](https://github.com/carryall/survey-challenge/pull/23#discussion_r530085962), I'm not sure that we should remove it since it's generated from angular when we generate a new component. If we remove it we need to keep removing it from every component we generate, what do you think?

Reply:
@carryall Usually we dont keep  empty space after last fuction/method of the class. Not sure if IDE/editor formatter can help or not by configuring  it so that it will remove similar empty spaces 🤔

Reply:
@carryall For this one I think we can remove it, according the `{}` has something inside that 🤔

Reply:
@Lahphim @junan removed in [e7e6610](https://github.com/carryall/survey-challenge/pull/23/commits/e7e661025ee4adc009d33c979a1c8e00468638d8)